packages/rest/src/rest.server.ts

```diff
@@ -211,6 +211,7 @@ export class RestServer
     config: RestServerConfig = {},
   ) {
     super(app);
+    this.scope = BindingScope.SERVER;
```

Comment:
I see we have to classes setting `scope = BindingScope.SERVER`:
- `RestServer`
- `ExpressServer`

In a typical REST application scaffolded via `lb4 app`, when I call `app.bind(...).inScope('BindingScope.SERVER')`, where will be the resolved value cached? At `RestServer` or `ExpressServer` level?

Reply:
It depends on where the binding is resolved. 

- If the binding is resolved from the RestServer or below, it will be cached using the `RestServerContext` as the key
- If the binding is resolved from the ExpressServer or below, it will be cached using the `ExpressServerContext` as the key

On the same application, it's possible that both RestServer and ExpressServer have a shared/cached value for the same binding.

Reply:
I thought that `RestServer` is using `ExpressServer` under the hood, that's why I was curious about where a value will be cached. I checked the sources and see that `ExpressServer` is not used by `RestServer` (or anywhere else in the typical LB4 stack), in which case my question is not relevant.

It does make me wonder though why are we providing & maintaining `ExpressServer` when we are not using it ourselves 🤷🏻